DynamicLights/src/generated/resources/.cache/af6b71d80a7bad2e79730e7a7b72cf91265d7018

```diff
@@ -0,0 +1,4 @@
+// 1.20.1	2023-11-11T03:45:54.0868385	Block States: dynamiclights
+3dff9e1ec59415779195d2abfa9f07bc17b428a1 assets/dynamiclights/blockstates/lit_air.json
```

Comment:
I have zero experience with these, should src/generated/ files be commited? It feels like ModDatagen would create the files

Reply:
Depends, some prefer to commit the files, some prefer to .gitignore and generate them locally.  I usually leave them in the repo so I can point people to what can be configured with a datapack.